source/slang/hlsl.meta.slang

```diff
@@ -19957,8 +19962,13 @@ ${
 
         case hlsl:
             uint isSingleLod = 0;
+            // Note: we cannot pass `Shape.dimensions`
+            // directly as a parameter into a Call, otherwise
+            // AST lookup fails (bug?), creating an intermediary works
+            // so we do this workaround.
+            int shapeDim = Shape.dimensions;
             Footprint footprint =  {__queryFootprint$(CoarseOrFine)NVAPI(
-                Shape.dimensions,
+                shapeDim,
```

Comment:
It seems like this may be due to the lookup scope of the expr isn't correct hooked up when you create the ctor call expr?

Reply:
This is very likely

Reply:
removed the WAR, and it doesn't crash anything.

Reply:
https://github.com/shader-slang/slang/pull/5107/commits/69cb426a73d67ddfdabcca87da4b49f544fcb507